Assets/Scripts/Source/Game/Combat/CombatManager.cs

```diff
@@ -1,12 +1,28 @@
 using UnityEngine;
 using System.Collections;
+using System.Collections.Generic;
 
 public class CombatManager : MonoBehaviour
 {
     // TODO: Replace with permanent solution
     [SerializeField]
     float delayBetweenActions = 1f;
 
+    [SerializeField]
```

Comment:
Can we refactor these into a configuration data class?

Reply:
Yes we can.